### ISSUE TITLE
Failed Custom Import leaves Ghost Model

### ISSUE THREAD

**Issue by kevinsaucier** [xLightsSequencer/xLights#4658]
Very minor 'bug' but, if a custom model fails to import, it leaves behind an invisible 'Custom' model.  Not sure if that causes other weird issues but just leaves garbage in the Model list.  Also not sure if this happens if a downloaded model fails to import as I couldn't find one that failed.

**Comment by derwin12** [comment]
Mind providing a step by step for this one?

**Comment by kevinsaucier** [comment]
I initiated an import, selected the xmodel file, and clicked ok.  I then got an error that the file was invalid and the import closed but left a model called Custom in the list.  Because it had no pixels, it was not visible on the layout.

A side note is that the reason I had an invalid xmodel file is that, when exporting the model, the file is written before a selection is made on whether or not you want to export the groups as well.  Apparently, after selecting groups (or no groups), something else is written to the xmodel file that makes it valid.  Seems like the file shouldn't even exist until it's able to be finalized with valid data.
